README.md

```diff
@@ -143,7 +144,7 @@ grunt watch
 npm link
 
 cd Foo
-npm install --save git://github.com/turn/Smart-Table.git#v0.3.8
+npm install --save git://github.com/turn/Smart-Table.git#v0.3.12
```

Comment:
Yeah that probably makes sense. I don't know how to change the README.tmpl.md to do that though. Do you?